packages/flame/test/components/mixins/gesture_hitboxes_test.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flame/collisions.dart';
+import 'package:flame/components.dart';
+import 'package:flame_test/flame_test.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group('GestureHitboxes', () {
+    testWithFlameGame('component with hitbox contains point', (game) async {
+      final component = _HitboxComponent();
+      component.position.setValues(1.0, 1.0);
+      component.anchor = Anchor.topLeft;
+      component.size.setValues(2.0, 2.0);
+
+      final hitbox = PolygonHitbox([
+        Vector2(1, 0),
+        Vector2(0, -1),
+        Vector2(-1, 0),
+        Vector2(0, 1),
+      ]);
+
+      component.add(hitbox);
+      await game.ensureAdd(component);
+
+      final point = component.position + component.size / 4;
+      expect(component.containsPoint(point), true);
```

Comment:
A test that tests whether `component.containsPoint` works if the hitbox is larger than the component and the `point` is outside of the component's size would be good to test.

Reply:
added